cypress/support/pageObjects/LandingPage.ts

```diff
@@ -0,0 +1,63 @@
+// 
+// Filename: LandingPage.ts
+// 
+
+export class LandingPage {
+  static getProjectName() {
+    return cy.get('[data-testid="projectName"]')
+  }
+  static getSWPRGuildCard() {
+    return cy.get('[data-testid=guildCard]').eq(2)
+  }
+
+  static getDXDGuildCard() {
+      return cy.get('[data-testid=guildCard]').eq(1)
+  }
+
+  static getREPGuildCard() {
+      return cy.get('[data-testid=guildCard]').eq(0)
+  }
```

Comment:
I think with this you are only testing that there are 3 guildCards. Since dev script doesn't ensure order `getDXDGuildCard` could be the REPguild card actually. Maybe you can refactor all `getXGuildCard` to look for the card containing the guild name you are looking for to be sure. Or testing only that `cy.get('[data-testid=guildCard]').length === 3`

Reply:
That's right, I can use that but I'm thinking to use assertion for every Guild card and 'cause of that I create different function